### PR TITLE
Randomless Kirby plants

### PR DESCRIPTION
**Pull request by Aylong220**
## What Does This PR Do
Adds static plants, since changing icon_state in the map editor, showed no effect.

## Why It's Good For The Game
Mappers will be able to beautifully arrange the plants they see fit, random ones will still be the same

## Images of changes
image

## Testing
<details>
 <summary>Yes</summary>

image
</details>

## Changelog
:cl:
tweak: Non-randomizable plants have been added for Mappers, players won't notice the difference
/:cl:

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -270,9 +270,10 @@
 
 /obj/item/kirbyplants/Initialize(mapload)
 	. = ..()
-	icon_state = "plant-[rand(1,35)]"
-	if(prob(1))
-		icon_state = "plant-36"
+	if(prob(1) && icon_state == "random_plant")
+		icon_state = "plant-eye"
+	if(icon_state == "random_plant")
+		icon_state = "plant-[rand(1,34)]"
```
Comment (Burzah) [inline code/game/objects/structures/flora.dm:276]:
```suggestion
		icon_state = "plant-[rand(1, 34)]"
```

File: code/game/objects/structures/flora.dm [hunk 2]
```diff
@@ -253,11 +253,11 @@
...
-
+// Kirby plants
 /obj/item/kirbyplants
 	name = "potted plant"
 	icon = 'icons/obj/flora/plants.dmi'
-	icon_state = "plant-1"
+	icon_state = "random_plant"
 	anchored = FALSE
 	layer = ABOVE_MOB_LAYER
 	w_class = WEIGHT_CLASS_HUGE
```
Comment (Sirryan2002) [inline code/game/objects/structures/flora.dm:263]: using `icon_state` to inform special behaviour is bad practice, make a `/obj/item/kirbyplants/random` subtype please

File: code/game/objects/structures/flora.dm [hunk 3]
```diff
@@ -314,10 +315,117 @@
...
+	icon_state = "plant-33"
+
+/obj/item/kirbyplants/plant34
+	icon_state = "plant-34"
+
+/obj/item/kirbyplants/applebush
+	icon_state = "applebush"
+
 /obj/item/kirbyplants/dead
+	icon_state = "plant-dead"
```
Comment (Sirryan2002) [inline code/game/objects/structures/flora.dm:424]: why diverge from the numbered scheme earlier? Is there something particularly different about this kirbyplants that set them apart from the rest?

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/flora.dm (hunk 3 above)
Comment (Aylong220) [reply code/game/objects/structures/flora.dm:424]: The apple bush was never as random as a dead plant

File: code/game/objects/structures/flora.dm (hunk 2 above)
Comment (Aylong220) [reply code/game/objects/structures/flora.dm:263]: In that case, it would be necessary to replace every plant on all the maps.....

Comment (Sirryan2002) [reply code/game/objects/structures/flora.dm:263]: a future TODO I guess then

**Review by S34NW — APPROVED** [review]

**Pull request merged**